scripts/rollup/packaging.js

```diff
@@ -94,6 +94,13 @@ async function copyAllShims() {
   await Promise.all([copyWWWShims(), copyRNShims()]);
 }
 
+async function copyReactReconcilerTypes() {
+  await asyncCopyTo(
+    require.resolve('react-reconciler/src/ReactFiberReconcilerTypes.js'),
```

Comment:
Why can't we put these in `react-reconciler/index.js.flow` in the first place?

Reply:
Sure we can. This PR is not ready to be merged yet. But I wanted to know if I'm headed the right way with the babel script.

Reply:
Tried explaining in the comment earlier :)